mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -1537,11 +1539,26 @@ struct Conv2DRewritePattern : public OpRewritePattern<T> {
     SmallString<8> gemmMPad_name("gemmMPad");
     SmallString<8> gemmNPad_name("gemmNPad");
 
+    int64_t nonGemmMSize = transformedFilterShape[1];
+    int64_t gemmMSize = transformedFilterShape[2];
     // filter pad start
-    // filter : K & CRS , if CRS is under 64 or 32
+    // K:output channel, C:input channel,R:filter height,S:filter width
```

Comment:
Would it be better if we use Y/X to represent filter width and height instead of R and S?